### PR TITLE
AOC-023: Day 6: Chronal Coordinates

### PR DESCRIPTION
**Pull request by basseur**
Day 06 has been completed!

### CODE REVIEW

**Comment by basseur** [comment]
Halt! Do not merge. There is a bug in the code. If the result is requested more than once, the Voronoi cells sizes are even further incremented!

**Comment by basseur** [comment]
The bug was fixed. But it is rather a workaround than a solution.

**Comment by basseur** [comment]
[AOC-023: Day 6: Chronal Coordinates](https://trello.com/c/ZiLjrGha/17-aoc-023-day-6-chronal-coordinates)

File: src/main/java/org/basseur/adventofcode/advent2018/days/day06/Day06.java [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+ * @author Jan Philipp G&ouml;lz
+ */
+@Component
+public class Day06 implements Days {
+
+    /** The location of the puzzle input file */
+    private static final String FILE_LOCATION = "src/main/java/org/basseur/adventofcode/advent2018/days/day06/Input.txt";
+    /** The puzzle status {@code HashMap} */
+    private final HashMap<String, ProblemStatusEnum> problemStatus;
+    /** The list of coordinate Strings */
```
Comment (mfbieber) [inline src/main/java/org/basseur/adventofcode/advent2018/days/day06/Day06.java]: Only a general comment, as I know some co workers might also have commented on this (I'm myself not a fan on being picky about this things in pull requests though): The name of your attribute says already what the comment says. So in a way, this is redundant information. 
Or: dos it get described in a JavaDoc afterwards if you comment it like this?

Comment (basseur) [reply src/main/java/org/basseur/adventofcode/advent2018/days/day06/Day06.java]: Yes, this is only for the JavaDocs. I don't know how useful JavaDocs really are in this project (probably not at all).

**Pull request merged**